package.json

```diff
@@ -2,7 +2,7 @@
   "author": "Transcend Inc.",
   "name": "@transcend-io/privacy-types",
   "description": "Core enums and types that can be useful when interacting with Transcend's public APIs.",
-  "version": "3.5.1",
+  "version": "4.0.0",
```

Comment:
Wondering why this merits a major version bump (as opposed to minor).

Reply:
@MadDataScience it's not really backwards compatible? If downstream clients were referring to this enum it would error out hard?

Reply:
Ah, makes sense.